alfresco-health-processor-platform/src/main/java/eu/xenit/alfresco/healthprocessor/plugins/solr/endpoint/AlwaysSearchEndpointSelector.java

```diff
@@ -0,0 +1,22 @@
+package eu.xenit.alfresco.healthprocessor.plugins.solr.endpoint;
+
+import java.util.Collections;
+import java.util.Set;
+import org.alfresco.service.cmr.repository.NodeRef.Status;
+
+/**
+ * Simple endpoint selector that selects a single endpoint for all nodes
+ */
+public class AlwaysSearchEndpointSelector implements SearchEndpointSelector {
+
+    private final SearchEndpoint searchEndpoint;
+
+    public AlwaysSearchEndpointSelector(String filter, SearchEndpoint searchEndpoint) {
```

Comment:
Yes, the parameter is completely useless. However, in the [SearchEndpointSelectorBeanPostProcessor](https://github.com/xenit-eu/alfresco-health-processor/pull/34/files#diff-d230fedb133bef1d07630ad8d5a82775910ce7071858c82f9e8e15cf95c08364R65), we put a filter parameter (we can't to it based on type, because the actual value not accessible in the postprocessor)